source/simulator/checkpoint_restart.cc

```diff
@@ -173,37 +152,151 @@ namespace aspect
 #endif
 
     }
+  }
+
+  template<int dim>
+  void Simulator<dim>::log_checkpoint(const std::string filename_for_triangulation, const std::string filename_for_serialization, bool is_quicksave)
+  {
+    std::ofstream checkpoint_log;
+    std::string checkpoint_file_name = parameters.output_directory + checkpoint_log_file;
+
+    if (!Utilities::fexists(checkpoint_file_name))
+      {
+        checkpoint_log.open(checkpoint_file_name, std::ios_base::out);
+        checkpoint_log << "Time_step_number filename_triangulation filename_mesh quicksave_slot_id" << std::endl;
```

Comment:
How about, 
 <time step number> <filename for triangulation> <filename for remaining state information> <quicksave slot id>

??

Not sure of the proper naming methodology here.